psi4/driver/molutil.py

```diff
@@ -283,7 +143,17 @@ def dynamic_variable_bind(cls):
     cls.__setattr__ = molecule_set_attr
     cls.__getattr__ = molecule_get_attr
 
-    cls.BFS = BFS
+    # for py3-only, all these `cls.fn = qcdb.Molecule._raw_fn` can be
+    #   replaced by `cls.fn = qcdb.Molecule.fn` and in qcdb/molecule.py
+    #   itself, the raw, staticmethod fns can use their official names again
+    #   and not need the append line at bottom of file. what I do for you,
+    #   py2 ...
```

Comment:
It might be worth considering dropping Py2 before too much longer. EOL for most major projects is end of this year, not the 2020 death clock.

Reply:
Guess we can twitter/GH/forum survey to see if there's major objection to dropping py27. The static/raw workaround wasn't so ugly as I had feared (I thought I'd have to have multiple fns with the same long args list). So while I've no affection for 27, I haven't yet reached the "Shoo!" stage.